docs/templates/resource-metadata.md

```diff
@@ -91,16 +93,17 @@ resource "coder_metadata" "resource_with_icon" {
 }
 ```
 
-To make easier for you to customize your resource we added some built-in icons:
+To make it easier for you to customize your resource we added some built-in
+icons:
 
 - Folder `/icon/folder.svg`
 - Memory `/icon/memory.svg`
 - Image `/icon/image.svg`
 - Widgets `/icon/widgets.svg`
 - Database `/icon/database.svg`
 
-We also have other icons related to the IDEs. You can see all the icons
-[here](https://github.com/coder/coder/tree/main/site/static/icon).
+We also have other icons related to the IDEs. You can see more information on
+how to use the builtin icons [here](./icons.md).
 
 ## Agent Metadata
```

Comment:
This Agent Metadata section is redundant. I suggest deleting it since it's already covered in `docs/templates/agent-metadata.md`